.github/setup-kryoptic.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# build kryoptic
+if [ ! -d "kryoptic" ]; then
+	git clone https://github.com/latchset/kryoptic.git
+fi
+pushd kryoptic
+git submodule init
+git submodule update
```

Comment:
nit: When we do dynamic build, we should not need to pull submodules.
Also please add a note to remove the custom build once we will have final package in Fedora.